tests/zquantum/core/circuits/_unitary_test.py

```diff
@@ -1,49 +1,55 @@
-import cirq
 import numpy as np
 import pytest
 import sympy
-from zquantum.core.circuits import (
-    RX,
-    RY,
-    RZ,
-    XX,
-    XY,
-    YY,
-    Circuit,
-    H,
-    I,
-    X,
-    Y,
-    Z,
-    export_to_cirq,
-)
+from zquantum.core.circuits import RX, RY, RZ, XX, XY, YY, Circuit, H, I, X, Y, Z
 
 
 class TestCreatingUnitaryFromCircuit:
     @pytest.mark.parametrize(
-        "circuit",
+        "circuit, expected_unitary_filename",
         [
             # Identity gates in some test cases below are used so that comparable
             # Cirq circuits have the same number of qubits as Zquantum ones.
-            Circuit([RX(np.pi / 5)(0)]),
-            Circuit([RY(np.pi / 2)(0), RX(np.pi / 5)(0)]),
-            Circuit([I(1), I(2), I(3), I(4), RX(np.pi / 5)(0), XX(0.1)(5, 0)]),
-            Circuit(
-                [
-                    XY(np.pi).controlled(1)(3, 1, 4),
-                    RZ(0.1 * np.pi).controlled(2)(0, 2, 1),
-                ]
+            (Circuit([RX(np.pi / 5)(0)]), "rx_gate.txt"),
+            (Circuit([RY(np.pi / 2)(0), RX(np.pi / 5)(0)]), "ry_rx_gate.txt"),
+            (
+                Circuit([I(1), I(2), I(3), I(4), RX(np.pi / 5)(0), XX(0.1)(5, 0)]),
+                "rx_xx_gate.txt",
+            ),
+            (
+                Circuit(
+                    [
+                        XY(np.pi).controlled(1)(3, 1, 4),
+                        RZ(0.1 * np.pi).controlled(2)(0, 2, 1),
+                    ]
+                ),
+                "xy_rz_gate.txt",
+            ),
+            (
+                Circuit([H(1), YY(0.1).controlled(1)(0, 1, 2), X(2), Y(3), Z(4)]),
+                "h_yy_x_y_z_gate.txt",
             ),
-            Circuit([H(1), YY(0.1).controlled(1)(0, 1, 2), X(2), Y(3), Z(4)]),
         ],
     )
-    def test_without_free_params_gives_the_same_result_as_cirq(self, circuit):
+    def test_without_free_params_gives_the_same_result_as_cirq(
+        self, circuit, expected_unitary_filename
+    ):
         zquantum_unitary = circuit.to_unitary()
-        cirq_circuit = export_to_cirq(circuit)
-        cirq_unitary = cirq.unitary(cirq_circuit)
+
+        """
+        Note: We decided to go with file-based approach after extracting cirq
+        from z-quantum-core and not being able to use `export_to_cirq` anymore.
```

Comment:
And why not just use SymbolicSimulator to calculate these unitaries?

Reply:
I also don't see why we have these in text files and not in a single file where we can import it from as a variable?
+ the path shouldn't be called `cirq_unitaries`, just `unitaries`, right?